.github/workflows/build.yml

```diff
@@ -87,28 +84,19 @@ jobs:
             vcvars_arch: "amd64_arm64"
             qt_ver: 6
             qt_host: windows
-            qt_arch: 'win64_msvc2019_arm64'
-            qt_version: '6.6.2'
-            qt_modules: 'qt5compat qtimageformats'
-            qt_tools: ''
+            qt_arch: "win64_msvc2019_arm64"
+            qt_version: "6.6.2"
+            qt_modules: "qt5compat qtimageformats"
+            qt_tools: ""
 
           - os: macos-12
             name: macOS
             macosx_deployment_target: 11.0
             qt_ver: 6
             qt_host: mac
-            qt_arch: ''
-            qt_version: '6.6.2'
-            qt_modules: 'qt5compat qtimageformats'
-            qt_tools: ''
-
-          - os: macos-12
```

Comment:
I think we should still have a macos-legacy build built for macos 10.14 as the target with Qt 6.4.3 (last version that still supported macOS 10.14 and 10.15)

Reply:
A good amount of people still use the macos-legacy build of Prism Launcher so I think we should still have it as an option for now.

Reply:
according to the download counts of release 8.0, it only made up ~8.35% of all macOS users. in comparison, we dropped windows legacy builds after 7.2 when they were ~2.88% of downloads (see the full stats [here](https://somsubhra.github.io/github-release-stats/?username=prismlauncher&repository=PrismLauncher&page=1&per_page=5#), might have got my math wrong lol). i don't think the difference here is super significant; and given that windows 8 support was dropped less than a year after it's EOL date, i would also say this follows precedent as 10.15 has been EOL for a year and a half now, with 10.14 slowly getting to 3

at the very least, we should be fairly ready to drop these versions when the time comes to bump our Qt version. i would just prefer to be a bit more proactive about it and bundle it with other deprecations rather than be forced to drop it later down the line

Reply:
You do have a point. I agree that we should be prepared to drop macs-legacy, but I say that we shouldn't rush it. I know macos-legacy does come at the cost of maintainence, but I think we should discuss it more before dropping it.

Reply:
If needed I can copy the ubuntu-20.04 but with 6.4.3